content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,277 @@
+
+### Registering new C++ tests
+
+While reading the upcoming steps, refer to the following `CMakeLists.txt` example:
+
+```
+# Preexisting module registration from completing the Prerequisite step
+ly_add_target(
+    NAME MyModuleName
+    NAMESPACE Gem
+    FILES_CMAKE
+        mymodule_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            Source
+        PUBLIC
+            Include
+    BUILD_DEPENDENCIES
+        PRIVATE
+            Legacy::CryCommon
+    COMPONENT
+        MyModuleComponent
+)
+
+# New test module registration from Step 1
+# This configures CMake to also build a library called `MyModuleName.Tests.so` (.dll on Windows)
+# The rest of CMake refers to this new module by its NAME ("MyModuleName.Tests")
+if(PAL_TRAIT_BUILD_SUPPORTS_TESTS)  # only create these test modules if the target platform supports testing
+
+    ly_add_target(
+        NAME MyModuleName.Tests SHARED
+        NAMESPACE Gem
+        FILES_CMAKE
+            mymodule_test_files.cmake
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                Tests
+                Source
+        BUILD_DEPENDENCIES
+            PRIVATE
+                AZ::AzTest
+                Legacy::CryCommon
+                Gem::MyModuleName
+        COMPONENT
+            MyModuleComponent
+    )
+
+    # Register new test module from Step 2
+    ly_add_googletest(
+        NAME MyModuleName.Tests
+    )
+
+endif()
+```
+
+#### Prerequisite: Add a production build target
+
+Before configuring tests, you must first define the library that you want to test. If your tests target an existing feature, then this step is already completed and you simply need to find the correct file path. Target libraries are defined in a `CMakeLists.txt`, which typically lives in the same directory as the code or in a parent directory. For more information on configuring `CMakeLists.txt` files, please refer to the [Build section](/docs/user-guide/build/). Note that you only need to define the production library that the tests target, not every feature in the production library. You can configure and write tests before you finish the production code.
+
+#### Step 1: Add a test build target
+
+Similar to the production build target, the test target defines a library in a `CMakeLists.txt` configuration file. Start by finding the `CMakeLists.txt` that you created in the prerequisite step.  It should exist at a path similar to `o3de/.../<MyModule>/CMakeLists.txt`.
+
+Modify the `CMakeLists.txt` file to define your new test module with `ly_add_target()`. Similar to the production build target, it's easiest to create another file that lists the C++ files used to compile the test library. The example above uses `o3de/.../<MyModule>/mymodule_test_files.cmake`, which has content similar to the following:
```

Comment:
Following up on this previous [comment](https://github.com/o3de/o3de.org/pull/1394/files/652f180c63330ac1259de5a222f35ca493002e53#r812424961), where we were looking to clarify that this instruction depends on the above **Prerequisite**, which links to the **[Build](https://deploy-preview-1394--o3deorg.netlify.app/docs/user-guide/build/)** section where it explains configuring `CMakeLists.txt` and creating `.cmake` files.

The confusion here is caused by a lack of info on `.cmake` files and their use in the Build section or anywhere in the O3DE documentation. This info is out-of-scope for this PR, but definitely needs to be addressed, so I'll file an issue for that. 

In the meantime, to clarify this step, simply stating ".cmake file" gives some hint and context. Also suggest putting last sentence in a new line to make it easier to scan descriptions of code samples. 
```suggestion
Modify the `CMakeLists.txt` file to define your new test module with `ly_add_target()`. Similar to the production build target, it's easiest to create another `.cmake` file that lists the C++ files used to compile the test library. 

The example above uses `o3de/.../<MyModule>/mymodule_test_files.cmake`, which has content similar to the following:
```